src/python/grpcio/grpc/_cython/.gitignore

```diff
@@ -1,4 +1,4 @@
-cygrpc.c
+cygrpc.cpp
```

Comment:
s/cpp/cc ?

Reply:
Cython only generates .cpp files (You can't tell it to generate cc files)

Also, I think you got dragged into this PR because I touched and then reverted a file you are an OWNER for, but I can't remove you as a reviewer (so you can ignore this pr)